### DISCUSSION TITLE
How to delete huge snapshots without disk space?

### DISCUSSION THREAD

**Discussion by davidfrickert** [longhorn/longhorn#7028] [Q&A]
Currently have a 12TB and 3TB snapshots due to disk expansion but have only 1TB left of storage in the disk.
Since from what I understand a snapshot deletion copies the data to the parent snapshot, is it even possible to delete such massive snapshots, or am I doomed to have to recreate the PVC to get a clean volume?

**Comment by jmcglock** [comment]
Seeing a similar issue here.
12 TB volume with almost 10 TB of "system generated" expand snapshots.
